src/core/errorHandler.js

```diff
@@ -31,41 +36,57 @@ export class ErrorHandler {
 
 class ErrorHandlerComponent extends React.Component {
   static propTypes = {
+    autoRenderErrors: PropTypes.boolean,
```

Comment:
Hah, okay, that makes sense but I only thought it through halfway and took `isRequired` in this case to be more/less a developer note that it requires a `defaultProp`. I should fix some of my components that use `isRequired` when they shouldn't, in that case. 😨